src/components/outreachMaterials/AssetCard.tsx

```diff
@@ -0,0 +1,75 @@
+import { OutreachAsset, OutreachTopic } from './formatted-type'
+import {
+  getYouTubeThumbnail,
+  getTopicImagePath,
+  truncateWithEllipsis,
+} from './utils'
+import { MediaResourceType } from '@/types/drupal/media'
+import { DownloadLink } from './DownloadLink'
+
+interface AssetCardProps {
+  asset: OutreachAsset
+  topics: OutreachTopic[]
+}
+
+export function AssetCard({ asset, topics }: AssetCardProps) {
+  const firstCategoryTopicId = asset.categories[0]
```

Comment:
Can `asset.categories` ever be undefined?

Reply:
No, it should fall back to an empty array if the data doesn't exist.